group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupCoordinatorRecordHelpersTest.java

```diff
@@ -108,12 +108,11 @@ public void testNewConsumerGroupMemberSubscriptionRecord() {
                 .setSupportedProtocols(protocols))
             .build();
 
-        CoordinatorRecord expectedRecord = new CoordinatorRecord(
-            new ApiMessageAndVersion(
-                new ConsumerGroupMemberMetadataKey()
-                    .setGroupId("group-id")
-                    .setMemberId("member-id"),
-                (short) 5),
+        CoordinatorRecord expectedRecord = CoordinatorRecord.record(
+            (short) 5,
```

Comment:
Given that we have constants defined for these values now, maybe the tests should exclusively use those instead of the numbers.